test/integrations/destinations/the_trade_desk/delivery/data.ts

```diff
@@ -0,0 +1,248 @@
+import {
+  destType,
+  destTypeInUpperCase,
+  advertiserId,
+  dataProviderId,
+  segmentName,
+  sampleDestination,
+} from '../common';
+
+beforeAll(() => {
+  process.env.THE_TRADE_DESK_DATA_PROVIDER_SECRET_KEY = 'mockedDataProviderSecretKey';
+});
+
+afterAll(() => {
+  delete process.env.THE_TRADE_DESK_DATA_PROVIDER_SECRET_KEY;
+});
```

Comment:
Not sure if `afterAll` & `beforeAll` make sense here. This is more of a `P1` issue or Non-blocking comment